UKHomeOffice/sgmr-service
# SPCR-145 Rename 'report' to 'voyage plan'

## Description
We have new user flows being tested with users. In the meantime, we should take the content on the prototype and update the existing pages on the service to match.

All user-visible mentions of "report" have been replaced by “voyage plan”.

## To Test
- Go to any page which would have previously contained "report"
- Ctrl + F in your browser and type "report"
- You should get no results
- Ctrl + F in your browser and type “voyage plan”
- You should get at least one result

## Developer Checklist

\* Required






Ensure you delete the branch once the Pull Request is merged.
